clippy_lints/src/from_instead_of_into.rs

```diff
@@ -74,7 +74,17 @@ impl LateLintPass<'tcx> for FromInsteadOfInto {
 
                         if !replace_trait_name.is_empty() && !target_trait_name.is_empty() {
                             let message = format!("{} trait is preferable than {} as a generic bound", replace_trait_name, target_trait_name);
-                            let sugg = format!("{}: {}<{}>", generic_arg_of_from_or_try_from, replace_trait_name, bounded_ty);
+                            let extracted_where_predicate = format!("{}: {}<{}>,", generic_arg_of_from_or_try_from, replace_trait_name, bounded_ty);
+                            let sugg;
+                            if wbp.bounds.len() == 1 {
+                                sugg = extracted_where_predicate;
+                            } else {
+                                let bounds: &[&GenericBound<'_>] = wbp.bounds.iter().filter(|b| ptr::eq(*b, target_bound)).collect();
+                                let bounds = bounds.map(|b| snippet(cx, b.span(), ".."));
+                                let first = bounds[0].to_string();
+                                let bounds_str = bounds[1..].fold(first, |s, &e| s + " + " + snippet(cx, e.span(), ".."));
```

Comment:
@flip1995 , I couldn't get this compile, what should I do?

Reply:
I don't think you have to make this that complicated. You don't need to include other bounds or the bounded type variable in the suggestion. Just use the span of the bound (`From`/`TryFrom`) and suggest to replace it with `Into`/`TryInto`

Reply:
@flip1995, I can't get what you mean, so if there is an `u32: Copy + From<T> + Clone` binding, if I extract `From<T>` out, what I think is to change it into `T: Into<u32>, u32: Copy + Clone`, how do you think I should handle this?

Reply:
Oh right, you have to swap the generic parameter... Yeah, that makes things more complicated. I have to test how to best do this myself.

Reply:
Thank you very much, @flip1995, I really don't know how to proceed.

Reply:
Thank you, @flip1995, after reading your suggestion, I did some research on `Span` and on `Iterator`, I think I can do this. 

But there are some part I still have doubt, when you refer to 2 part suggestion you are not referring to multi span suggestion, do I understand correctly?